test/test_cuda.py

```diff
@@ -1032,6 +1033,23 @@ def test_var_stability(self):
         tensor = tensor.unsqueeze(1)
         self.assertEqual(tensor.var(0)[0], 0.03125)
 
+    def test_digamma(self):
```

Comment:
Any reason why `digamma` can't be tested like all other functions?

Reply:
The values of digamma on gpu when compared to cpu are the same to 6 digits. I tried making this a normal test and some output values were high (~100), resulting in an error of around 1e-3, which is above the error threshold